Jenkinsfile

```diff
@@ -24,6 +24,7 @@ pipeline {
         IMAGE_ES_TAG = '7.10.1'
         // Only Python image tag changes for every build
         PYTHON_IMAGE_TAG = "${JOB_BASE_NAME}-${BUILD_NUMBER}"
+        FUNCTIONAL_TESTS_URL = "http://dev-jenkins/job/cf.gov-functional-tests/${BUILD_NUMBER}/consoleFull"
```

Comment:
I think we're getting close. This change gives a URL of http://dev-jenkins/job/cf.gov-functional-tests/12/consoleFull for the checks in this PR. But the `12` that gets inserted in the `${BUILD_NUMBER}` spot looks like it's referring to the build number for the PR-6775 pipeline job, not the build number for the functional tests (looks like 1295 in this case), which is what we'd need. I don't know if that's an easy build number to get, but if not, it might not be worth pursuing this one much more.

Reply:
ah shoot, yeah the other build number isn't determined at the Jenkins job creation time. I could have it point to just the functional tests jobs list, and there the user can select their running job. It will technically not link to Ocean view anymore, but I guess it would still be confusing if two people are running a job at the same time.